cmd/cosign/cli/sign.go

```diff
@@ -166,7 +167,12 @@ EXAMPLES
 			if len(args) == 0 {
 				return flag.ErrHelp
 			}
-
+			switch *attachment {
+			case "none", "sbom":
```

Comment:
What do you think of keeping a list of these near https://github.com/sigstore/cosign/blob/main/pkg/cosign/fetch.go#L57? That way we could verify against it.